package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Can't say about this. Let's ask @jywarren about it.

Reply:
Its still there, could you please try one more time to remove it

Reply:
Hiya. In my forked one the package lock is gone and the pull request seems to be saying everything in it is gone. Am I reading it wrong?